src/aio/poller_kqueue.inc

```diff
@@ -107,6 +105,7 @@ void nn_poller_reset_in (struct nn_poller *self, struct nn_poller_hndl *hndl)
     if (hndl->events & NN_POLLER_EVENT_IN) {
         EV_SET (&ev, hndl->fd, EVFILT_READ, EV_DELETE, 0, 0, 0);
         rc = kevent (self->kq, &ev, 1, NULL, 0, NULL);
+        nn_assert (rc == 0);
```

Comment:
It's definitely wrong; it's not a good fix, but a placeholder for something that needs to be further investigated. The compiler warned of an unused assignment to `rc` here -- and for `kevent`, we don't even expect `rc == 0`